demo/src/app/components/alert/demos/basic/basic.alert.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+import { NgbAlert } from '@ng-bootstrap/alert';
```

Comment:
@icfantv did this import work for you? I'm not sure how it could work since we didn't publish anything to npm...